


from mysql.connector import Error
from entities.base import Base